packages/react-charting/src/components/Legends/Legends.base.tsx

```diff
@@ -161,10 +174,21 @@ export class LegendsBase extends React.Component<ILegendsProps, ILegendState> {
    * @param legend ILegend
    */
   private _canSelectMultipleLegends = (legend: ILegend): void => {
-    const selectedLegends = silceOrAppendToArray(this.state.selectedLegends, legend.title);
+    let selectedLegends = { ...this.state.selectedLegends };
+    if (selectedLegends[legend.title]) {
+      // Delete entry for the deselected legend to make
+      // the number of keys equal to the number of selected legends
+      delete selectedLegends[legend.title];
+    } else {
+      selectedLegends[legend.title] = true;
```

Comment:
This will mean that legend.title is the last legend to be selected. In that case resetting the legends should not happen?